src/widgets/training-program/ui/training-program.module.scss

```diff
@@ -0,0 +1,91 @@
+.training-program {
+  &.column-2 {
+    display: flex;
+    flex-direction: row;
+    gap: 100px;
+    align-items: center;
+    justify-content: flex-start;
```

Comment:
Not sure, but maybe we could use the .column-2 global class here and override just a few styles that are different? Because a lot of properties are repeated

Reply:
removed